src/Rector/Contrib/PHPUnit/ScenarioToTestAnnotationRector.php

```diff
@@ -0,0 +1,51 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Rector\Contrib\PHPUnit;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt\ClassMethod;
+use Rector\Rector\AbstractPHPUnitRector;
+use Rector\ReflectionDocBlock\NodeAnalyzer\DocBlockAnalyzer;
+
+/**
+ * Before:
+ * - @scenario
+ *
+ * After:
+ * - @test
+ */
+final class ScenarioToTestAnnotationRector extends AbstractPHPUnitRector
```

Comment:
A whole Rector for changing an annotation? Can't we create a Dynamic one, like `ClassReplacerRector`? :thinking:

Reply:
I think we can.

How would you differ `@scenario` in Codeception and `@scenario` in PHPUnit?

Reply:
We can receive a class, and look at it parents, and the annotation name, like:
```yml
PHPUnit\Framework\TestCase:
   - 'test': 'scenario'
```

Reply:
That could work!
And how would you limit annotations to nodes? To `ClassMethod` and `Property` nodes or all of them?

I'll try to handle this later